### ISSUE TITLE
UI frozen, no obvious link to user action

### ISSUE THREAD

**Issue by AustinC** [KronicDeth/intellij-elixir#2408]
**Describe the bug**
UI frozen, no obvious link to user action. Forced termination of IDE.
Zipped logs attached.

**Logs**
[threadDumps-freeze-20211220-164936-IU-213.5744.223.zip](https://github.com/KronicDeth/intellij-elixir/files/7748794/threadDumps-freeze-20211220-164936-IU-213.5744.223.zip)

**Desktop:**
```
 ~ # sw_vers
ProductName:    macOS
ProductVersion: 11.5.2
BuildVersion:   20G95
```

**Erlang:**
- Installer [e.g. homebrew]
- Version [e.g. 21]

**Elixir:**
- Installer [e.g. kiex]
- Version [e.g. 1.8]

**Plugin:**
v 12.0.1

**Comment by the-destro** [comment]
I'm having the same issue. m1 mac. Downgrading to 11.13 is the only workaround

**Comment by KronicDeth** [comment]
In the dumps you posted @AustinC I don't see anything obvious. It looks like a problem with `UnaliasedName` maybe.  @the-destro without your specific logs I can't confirm your freeze has the same source.

**Comment by KronicDeth** [comment]
@the-destro what version of the plugin are you on now?  Performance and freeze fixes related to over-decompiling were in 12.2.1, so I should have closed this then.  If you tell me you're on 13.0.0 I'll move your latest comment to a new issue.
